src/components/RegisterUser/RegisterUser.js

```diff
@@ -0,0 +1,22 @@
+import React, { useEffect } from 'react'
+import { get } from 'lodash/fp'
+
+export function RegisterUser (props) {
```

Comment:
This component is mostly a placeholder. I expect our auth/login requirements to be more complicated than this. But for now, we have to make sure that the hostingUser is registered as a host or they can't see the available happs.